### PR TITLE
[7] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by kazeemerinfolami**
## Task: `[7] Develop a Contact Form Page using HTML`
- `Contact Form`
Screen Shot 2020-09-20 at 9 13 54 PM

### CODE REVIEW

File: submissions/kazeemErinfolami/html/index.html [hunk 1]
```diff
@@ -11,8 +11,8 @@ <h1><a href="#">Emarket</a></h1>
       </div>
       <nav>
         <ul>
-          <li><a href="#">Cart</a></li>
-          <li><a href="#">Sign In</a></li>
+          <li><a href="../htmlForms/contact.html">Contact Form</a></li>
+          <li><a href="../htmlForms/signUp.html">SignUp</a></li>
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/html/index.html:15]: Do you have a signup page?

Comment (kazeemerinfolami) [reply submissions/kazeemErinfolami/html/index.html:15]: Yes, but thought you asked me to remove it

Comment (kazeemerinfolami) [reply submissions/kazeemErinfolami/html/index.html]: Hope everything is fine here, can I move on to the next project?

File: submissions/kazeemErinfolami/htmlForms/contact.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  </head>
+  <body>
+    <nav>
+      <ul>
+        <li><a href="../html/index.html">Home</a></li>
+      </ul>
+    </nav>
+    <form>
+      <div>
+        <h1>Contact Form</h1>
```
Comment (ToluAkin) [inline submissions/kazeemErinfolami/htmlForms/contact.html:16]: header elements are block elements. You don't have to add it with a div if there is no extra use of the div for the header @kazeemerinfolami You can remove the div tags

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/kazeemErinfolami/html/index.html [hunk 3]
```diff
@@ -11,8 +11,7 @@ <h1><a href="#">Emarket</a></h1>
       </div>
       <nav>
         <ul>
-          <li><a href="#">Cart</a></li>
-          <li><a href="#">Sign In</a></li>
+          <li><a href="../htmlForms/contact.html">Contact Form</a></li>
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/html/index.html:14]: Linking this done in the next task. you can remove it

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**